content/_guides/cloaks.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Cloaks 
+category: using
+---
+
+A cloak is a custom hostmask that obscures your client hostname or IP address,
+as shown in IRC operational messages or `/whois`. 
+
+For example, the hostmask `jsmith@10-10-32-1.res.dsl.example.com` might appear
+with a cloak as `jsmith@user/jsmith`.
+
+A cloak can help other users and bots identify you simply and consistently. It
+can also help limit the exposure of your IP address on the network, although
+it does not necessarily provide anonymity or prevent the discovery of your IP
+address. 
+
+As an individual, you can get a generic user cloak in the form of
+`user/username`. If you are involved with a well-known project or community,
+you may be able to get a cloak that indicates your affiliation with that
+group. 
+
+## Generic user cloaks
+
+You can request a generic user cloak from the bot in `#libera-cloak`. The
+cloak will be in the form of `user/<NickServ username>`. Registration with
+`NickServ` is required.
+
+### To request a cloak from the bot:
+
+1. [Register with NickServ](/guides/registration) if you have not already
+   done so.
+2. `/join #libera-cloak`
+3. Say `!cloakme`
+
+The bot will give your account a generic user cloak and then — because its work
+is done — politely kick and ban you from the channel. Your new cloak takes
+effect immediately. 
+
+## Project or community cloaks
+
+If you are involved with a well-known project or community, you may request a
+cloak indicating your affiliation. For help with this, ask someone associated
+with your project or community who can coordinate with Libera.Chat to
+administer cloaks.
```

Comment:
we should probably specify that "group contacts" are the people who can request project cloaks

Reply:
made an attempt at this